### DISCUSSION TITLE
AtroCore web interface not accessable via URL after installation

### DISCUSSION THREAD

**Discussion by markus4church** [atrocore/atrocore#809] [Q&A]
Dear AtroPIM-community,

I have a problem to access the installation wizard for my AtroCore Application in the web interface from my project URL,
the last step of installation guide.

Installation went well so far, according to step-by-step installation guide including setup Linux Preparation.
Just did some small deviations from manual e.g. ensure root access with 'sudo -s'  for configuring PHP etc. 

On configuring the vitual host, for my_domain I just invented one, e.g. 'abc-ek.de'.
In web-browser tying 'localhost' or typing <my_IP> brings me to the "Hello World" landing page.

However at the end of installation guide procedure, when entering my domain 'abc-ek.de' to web browser, it doesn't lead to the web wizard of AtroCore.
I already added in /etc/apache2/apache2.conf below the “# Global configuration” section the "ServerName <my_IP>" entry and restarted the apache server. command "sudo apache2ctl configtest" returned 'Syntax OK'.
Also the config-file with VirtualHost was created, enabled and apache reloaded.

Are there further things to consider?

The apache server is up and running, also the installation of mysql database went without error. 

System used:
- VM managed via Hyper-V
- Ubuntu 22.04 LTS
- Apache Server 
- PHP 8.1.2-1ubuntu2.13

**Comment by exeqtwasnotfree** [comment]
Do you have DNS in place?
If you visit 'abc-ek.de' in web-browser, do you get the hello world page?
To me it sounds what you may set up the site correct, but your client needs to know where abc-ek.de are located.
If the domain is not resolved by DNS, you have to locally map domain to IP by edit the host file.

**Comment by markus4church** [comment]
Thanks a lot. After mapping the domain in the host file locally, I come to the "hello world" page via the domain now, not only by typing the IP in the web-browser. 
But how to start the atrocore-wizard in the webpage? 
Was hoping to see the wizard after typing 'abc-ek.de' in web-browser, 
like one can see around Second 50 in this video on youtube: https://youtu.be/F2OP5aUMsVM 
I did a restart on apache web server, but still see the "hello world" page. 
What did I miss?
thx

**Comment by exeqtwasnotfree** [comment]
If you now get the hello page, I guess the webserver shows index.html instead of index.php
If you delete index.html or index.htm in the webserver root-folder, there should be files left similar to the files showing in the movie at 22 seconds.
Try the domain again after checking this.

**Comment by markus4church** [comment]
Thanks again for the information!
Now it works, and Atrocore wizard also worked fine to all the end until configuration of data model can start now. 
Pretty exited!
